README.md

```diff
@@ -1,2 +1,28 @@
-# static-code-analysis-tool
-Tool for performing static code analysis for Ballerina projects.
+# Ballerina Static Code Analysis Tool
+
+## Overview
+
+Static Code Analysis (SCA) is an approach that uses tools to examine code without execution. They are used for identifying potential issues like bugs, vulnerabilities, and style violations. SCA improves software quality by early issue detection, ensuring better maintainability, and providing enhanced security.
+
+The Ballerina scan tool is a Ballerina tool that provides the command-line functionality to statically analyze Ballerina files and report analysis results. The scan tool consists of core scan logic and extension points for introducing additional analysis and reporting results to static code analysis platforms.

```

Comment:
Introduce this as Ballerina supports SCA using the Ballerina scan tool ....

Then instead of the second sentence, on a new line, let's say

This repository consists of 
- the Ballerina scan tool implementation
- the core scan logic
- extension points for ....